src/Controller/Admin/Document/DocumentController.php

```diff
@@ -1056,8 +1056,8 @@ public function diffVersionsAction(Request $request, int $from, int $to, Documen
         }
 
         try {
-            Chromium::convert($prefix . $router->generate('pimcore_admin_document_document_diff_versions_html', ['id' => basename($fromHtmlFile)]), $fromImageFile);
-            Chromium::convert($prefix . $router->generate('pimcore_admin_document_document_diff_versions_html', ['id' => basename($toHtmlFile)]), $toImageFile);
+            HtmlToImage::convert($prefix . $router->generate('pimcore_admin_document_document_diff_versions_html', ['id' => basename($fromHtmlFile)]), $fromImageFile);
```

Comment:
Ideally, we could create a new method in `11.2` eg. `convertByHtml` and pass the string `$docContentFrom` and change in the core to use [$page->setHtml()](https://github.com/chrome-php/chrome?tab=readme-ov-file#set-the-page-html) instead of [navigate](https://github.com/pimcore/pimcore/pull/16608/files#diff-74767dcb230d5e59d6f91308ab2f91896c04ea2a15292c88915745eefb12f247R189)

Should be quicker and avoid temp files

Reply:
Leaving for future, so far works